test/test_common/network_utility.cc

```diff
@@ -116,10 +116,24 @@ Address::InstanceConstSharedPtr getCanonicalLoopbackAddress(Address::IpVersion v
   return Network::Utility::getIpv6LoopbackAddress();
 }
 
-Address::InstanceConstSharedPtr getAnyAddress(const Address::IpVersion version) {
+// There is no portable way to initialize sockaddr_in6 with a static initializer, do it with a
+// helper function instead
+static sockaddr_in6 v6any_() {
```

Comment:
Envoy convention is to put this in the anonymous namespace rather than mark it `static`. Might be clearer to rename it to `sockaddrIn6Any()` to match Envoy convention.

Reply:
Applied.